Source/CesiumRuntime/Private/CesiumTile.cpp

```diff
@@ -0,0 +1,81 @@
+#include "CesiumTile.h"
+#include "Components/PrimitiveComponent.h"
+#include "VecMath.h"
+
+namespace {
+struct GetBoundsOperation {
```

Comment:
This looks copied from `CalcBounds.cpp`. Can we use that instead of copying it? Perhaps with minor changes.